### PR TITLE
Minor typos, added moar messengers

### PR DESCRIPTION
**Pull request by wprzyb**

### CODE REVIEW

File: pasta_pl.txt [hunk 1]
```diff
@@ -80,4 +80,24 @@ Albo Die With Me
...
+Albo Semaphora
+Albo stealthy.im'a
+Albo Fortknoxtera
+Albo Crypvisera
+Albo Tungstena
+Albo ȱ
+Albo vero
+Albo tinode'a
+Albo Dust'a
+Albo ZeroMQ
```
Comment (mciancia) [inline pasta_pl.txt:95]: Does ZeroMQ count as communicator? Afair it's just a library/protocol

**Review by mciancia — CHANGES_REQUESTED** [review]

File: pasta_pl.txt (hunk 1 above)
Comment (wprzyb) [reply pasta_pl.txt:95]: I'm a simple dude, I see something at [digital communications protocol google sheet](https://docs.google.com/spreadsheets/d/1-UlA4-tslROBDS9IqHalWVztqZo7uxlCeKPQ-8uoFOU/edit), I copy it here.

Comment (wprzyb) [reply pasta_pl.txt:95]: And I'm not alone in this, major part of this list has entries from that sheet in the very same order in some parts

Comment (propiro) [reply pasta_pl.txt:95]: this conversation is opressing, and break CoC that will never exist here. Pull request accepted with dusgust.

**Pull request merged**